core/modules/layout/tests/layout.test

```diff
@@ -1190,3 +1190,179 @@ class LayoutUpgradePathTest extends UpgradePathTestCase {
     return $this->assertBlocks($blocks, TRUE);
   }
 }
+
+/**
+ * Tests invocation of hooks when performing an action.
+ *
+ * Tested hooks are:
+ * - hook_layout_presave
+ * - hook_layout_insert
+ * - hook_layout_update
+ * - hook_layout_disable
+ * - hook_layout_enable
+ * - hook_layout_delete
+ * - hook_layout_block_remove
+ * - hook_layout_template_change
+ * - hook_layout_revert
+ */
+class LayoutHookTestCase extends BackdropWebTestCase {
+  protected $admin_user;
+
+  function setUp() {
+    parent::setUp('layout_test');
```

Comment:
If you switch to the testing profile, this will need to explicitly turn on the layout module:

```
parent::setUp('layout', 'layout_test');
```